### PR TITLE
Fixed typo in build_activation_images.py

### PR DESCRIPTION
**Pull request by Sai-Pra**
In line 24 of build_activation_images.py, I changed "programmaticly" to "programmatically" to be dramatically correct.

### CODE REVIEW

**Comment by Sai-Pra** [comment]
Hi there for anyone viewing this commit! This is my actually first commit to the PyTorch project (although insignificant, I hope it's the first of many)!

As of the time viewing this, this pull request is saying it "restricts merging to authorized users" and "requires approval from a maintainer". Are those just the PyTorch admins?

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**
